consensus/bor/bor.go

```diff
@@ -355,7 +352,12 @@ func (c *Bor) verifyHeader(chain consensus.ChainHeaderReader, header *types.Head
 	isSprintEnd := IsSprintStart(number+1, c.config.CalculateSprint(number))
 
 	// Ensure that the extra-data contains a signer list on checkpoint, but none otherwise
-	signersBytes := len(header.Extra) - extraVanity - extraSeal
+	var signersBytes int
+	if c.config.IsParallelUniverse(header.Number) {
+		signersBytes = len(header.GetValidatorBytes())
+	} else {
+		signersBytes = len(header.Extra) - types.ExtraVanityLength - types.ExtraSealLength
+	}
```

Comment:
Can we move this if..else to `header.GetValidatorBytes` and add block # as a parameter to the function? The pattern seems to be repeated in multiple places.

Reply:
[Done](https://github.com/maticnetwork/bor/pull/930/commits/646a421232dcfedfdcd1731e932f19455f16cc84)